src/custom/components/swap/FeeInformationTooltip.tsx

```diff
@@ -48,16 +52,19 @@ export default function FeeInformationTooltip(props: FeeInformationTooltipProps)
           text={
             <div>
               <FeeTooltipLine>
-                <span>Before fees</span>
+                <span>Before fee</span>
                 <span>{amountBeforeFees}</span>{' '}
               </FeeTooltipLine>
               <FeeTooltipLine>
-                <span>GP fee</span>
-                <span>{feeAmount}</span>{' '}
+                <span>Fee</span>
+                <span>
+                  {type === 'From' ? '+' : '-'}
+                  {feeAmount}
+                </span>{' '}
               </FeeTooltipLine>
               <FeeTooltipLine>
                 <span>Gas costs</span>
-                <strong>Free</strong>{' '}
+                <strong className="green">Free</strong>{' '}
```

Comment:
I'd omit the verboseness of adding the `className` (unless we rely on it for a function?). 

Instead, given this is specific to `<FeeTooltipLine>` *and* `<strong>` is the only direct nested element, I'd add it to its Style Component like this:

const FeeTooltipLine = styled.p`
  > strong {
    color: ${({ theme }) => theme.green1};
  }
`

Reply:
> this would target all strongs but good call on themes

All direct children (>) only, and my comment took into consideration the likelihood of other direct 'competing' child `<strong>` elements to exist here (very low and if we must, we'd use another element). 

In general a fan, because it keeps code dry, without sacrificing semantic readability. 

This falls apart if we have bigger/more complex components, I'm aware.

Reply:
Sorry, just opened the expanded code view...

I see now. Could get away with a `<b>` heh. 

In general would you prefer a className vs. a specific styled component just for the green `<strong>` element?

Reply:
i don't like the `className` approach mixed with styled, but in this case as a one-off i don't really care tbh. but if you feel strongly against it i can rework it

Reply:
Not `<stronk>` perse, your call otherwise. I think my thinking is more, to keep the markup in the component as much as possible class-less and strictly only use styled components and or 'naked' HTML elements (to be targeted with styled components).